clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java

```diff
@@ -667,7 +667,16 @@ protected void handleTagCommand() {
         if (propertySet != null && valueSet != null) {
           newSetTagProperty(url, ignore, metalake, getOneTag(tags), propertySet, valueSet).handle();
         } else if (propertySet == null && valueSet == null) {
+          if (!name.hasName()) {
+            System.err.println(ErrorMessages.MISSING_NAME);
+            Main.exit(-1);
+          }
           newTagEntity(url, ignore, metalake, name, tags).handle();
+        } else {
+          System.err.println(
+              "Command cannot be executed. The set command only supports configuring tag properties or attaching "
+                  + "tags to entity.");
+          Main.exit(-1);
```

Comment:
This is a bit long and not in teh same style as other error messages. I'd shorten it to "The set command only supports tag properties or attaching tags."